chromium_src/chrome/common/pref_names.cc

```diff
@@ -0,0 +1,14 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/common/pref_names.h"
+
+#if BUILDFLAG(IS_WIN)
```

Comment:
you need to include build_config.h for this, but you can drop this guard for the pref, it's not really necessary

Reply:
done

Reply:
it's fine to leave it, but it's just one more thing you have to check for a correct guard